src/main/java/com/esri/samples/na/routing_around_barriers/README.md

```diff
@@ -0,0 +1,91 @@
+<h1>Routing around barriers</h1>
+
+<p>Find a route that reaches all stops without crossing any barriers.</p>
+
+<p><img src="RoutingAroundBarriers.png"/></p>
+
+<h2>Use case</h2>
+
+<p>You can define barriers to avoid unsafe areas, for example flooded roads, when planning the most efficient route to evacuate a hurricane zone. When solving a route, barriers allow you to define portions of the road network that cannot be traversed. You could also use this functionality to plan routes when you know an area will be inaccessible due to a community activity like an organized race or a market night.
```

Comment:
Thanks, that would be a good point to make in the readme formatting documentation. My approach so far was to describe the feature in a general, technical way, then move on to explain what use case it may apply to.